### PR TITLE
make java-version an input

### PR DESCRIPTION
**Pull request by jetersen**

### CODE REVIEW

File: .github/workflows/maven-cd.yml [hunk 1]
```diff
@@ -53,11 +59,11 @@ jobs:
       with:
         fetch-depth: 0
 
-    - name: Set up JDK 8
+    - name: Set up JDK ${{ inputs.java-version }}
```
Comment (jglick) [inline .github/workflows/maven-cd.yml:62]: or just #11

**Review by jglick — COMMENTED** [review]
:shrug: mild preference for just hard-coding 11 (the status quo in `main` if not `v1`). Once we have plugins start to pick up post-switch core versions, if any overrode this to say 8, their releases would start to break.

File: .github/workflows/maven-cd.yml (hunk 1 above)
Comment (jetersen) [reply .github/workflows/maven-cd.yml:62]: > Once we have plugins start to pick up post-switch core versions, if any overrode this to say 8, their releases would start to break.

Fine #11 wins 😅

**Comment by jglick** [comment]
I guess we just pay attention to whether there are any serious issues with releasing from JDK 11. So far I have heard about one relatively minor bug in Javadoc; and a Javadoc frameless mode which some people like and some people dislike (but the opinion of the plugin _maintainer_ is not particularly relevant since they are unlikely to be the ones viewing their own Javadoc).

**Pull request closed without merging**